docs/gitrepo-add.md

```diff
@@ -168,6 +168,39 @@ For example, to add a secret in kubectl, run
 
 After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Make sure secret is created under the same namespace with gitrepo.
 
+### Use different helm credentials for each path
+
+:::info
+`gitRepo.spec.helmSecretName` will be ignored if `gitRepo.spec.helmSecretNameForPaths` is provided
+:::
+
+Create a file `secrets-path.yaml` that contains credentials for each path defined in a `GitRepo`. Credentials will not be used
```

Comment:
This line ends with what looks like an incomplete sentence; a leftover from an edit maybe? :thinking:

Reply:
ohh good catch! It broke when I applied https://github.com/rancher/fleet-docs/pull/69#discussion_r1206849784 
fixed in https://github.com/rancher/fleet-docs/pull/69/commits/600dd350f51e30ba94e308be934417cd4867d14b